--- proto/api/trading.proto
+++ proto/api/trading.proto
@@ -23,10 +23,10 @@ service trading {
   rpc PrepareAmendOrder(AmendOrderRequest) returns (PrepareAmendOrderResponse);

   // Request balance increase
-  rpc NotifyTraderAccount(NotifyTraderAccountRequest) returns (NotifyTraderAccountResponse);
+  // rpc NotifyTraderAccount(NotifyTraderAccountRequest) returns (NotifyTraderAccountResponse);

   // Request withdrawal
-  rpc Withdraw(WithdrawRequest) returns (WithdrawResponse);
+  // rpc Withdraw(WithdrawRequest) returns (WithdrawResponse);

   // Submit a signed transaction
   rpc SubmitTransaction(SubmitTransactionRequest) returns (SubmitTransactionResponse);
@@ -49,21 +49,21 @@ message SubmitTransactionResponse {
   bool success = 1;
 }

-message WithdrawRequest {
-  vega.Withdraw withdraw = 1;
-}
+// message WithdrawRequest {
+//   vega.Withdraw withdraw = 1;
+// }

-message WithdrawResponse {
-  bool success = 1;
-}
+// message WithdrawResponse {
+//   bool success = 1;
+// }

-message NotifyTraderAccountRequest {
-  vega.NotifyTraderAccount notif = 1;
-}
+// message NotifyTraderAccountRequest {
+//   vega.NotifyTraderAccount notif = 1;
+// }

-message NotifyTraderAccountResponse {
-  bool submitted = 1;
-}
+// message NotifyTraderAccountResponse {
+//   bool submitted = 1;
+// }

 message PrepareSubmitOrderResponse {
   bytes blob = 1;
--- proto/vega.proto
+++ proto/vega.proto
@@ -279,16 +279,16 @@ enum ChainStatus {
     CONNECTED = 2;
 }

-message NotifyTraderAccount {
-  string traderID = 1;
-  uint64 amount = 2;
-}
+// message NotifyTraderAccount {
+//   string traderID = 1;
+//   uint64 amount = 2;
+// }

-message Withdraw {
-  string partyID = 1;
-  uint64 amount = 2;
-  string asset = 3;
-}
+// message Withdraw {
+//   string partyID = 1;
+//   uint64 amount = 2;
+//   string asset = 3;
+// }

 message OrderAmendment {
   // required to find the order, not being updated
