--- proto/api/trading.proto
+++ proto/api/trading.proto
@@ -25,7 +25,7 @@ service trading {
   rpc SignIn(SignInRequest) returns (SignInResponse);
 
   // Request balance increase
-  rpc NotifyTraderAccount(NotifyTraderAccountRequest) returns (NotifyTraderAccountResponse);
+  // rpc NotifyTraderAccount(NotifyTraderAccountRequest) returns (NotifyTraderAccountResponse);
 
   // Request withdrawal
   rpc Withdraw(WithdrawRequest) returns (WithdrawResponse);
@@ -42,13 +42,13 @@ message WithdrawResponse {
   bool success = 1;
 }
 
-message NotifyTraderAccountRequest {
-  vega.NotifyTraderAccount notif = 1;
-}
+// message NotifyTraderAccountRequest {
+//   vega.NotifyTraderAccount notif = 1;
+// }
 
-message NotifyTraderAccountResponse {
-  bool submitted = 1;
-}
+// message NotifyTraderAccountResponse {
+//   bool submitted = 1;
+// }
 
 message SignInRequest {
 
--- proto/vega.proto
+++ proto/vega.proto
@@ -271,10 +271,10 @@ message PendingOrder {
   Order.Type type = 10;
 }
 
-message NotifyTraderAccount {
-  string traderID = 1;
-  uint64 amount = 2;
-}
+// message NotifyTraderAccount {
+//   string traderID = 1;
+//   uint64 amount = 2;
+// }
 
 message Withdraw {
   string partyID = 1;
